lib/nerves_hub_link/configurator.ex

```diff
@@ -48,7 +48,9 @@ defmodule NervesHubLink.Configurator do
       |> Keyword.put_new(:verify, :verify_peer)
       |> Keyword.put_new(:server_name_indication, to_charlist(base.device_api_sni))
 
-    %{base | params: Nerves.Runtime.KV.get_all_active(), socket: socket, ssl: ssl}
+    params = Map.put(Nerves.Runtime.KV.get_all_active(), "nerves_fw_fwup_version", fwup_version())
```

Comment:
```suggestion
    params = Map.put(Nerves.Runtime.KV.get_all_active(), "fwup_version", fwup_version())
```

I'm not sure that the `nerves_fw_` prefix matters here. My preference would be to drop it, but I don't feel strongly about it.